src/ucp/api/ucp.h

```diff
@@ -1299,6 +1310,61 @@ ucs_status_t ucp_ep_flush(ucp_ep_h ep);
  * by a user, then the user is responsible for segmentation and subsequent
  * management.
  *
+ * <table>
+ * <caption id="ucp_mem_map_matrix">Matrix of behavior</caption>
+ * <tr><th>Parameter/Flag                  <th colspan="8">Value
+ * <tr><td>@ref UCP_MEM_MAP_NONBLOCK       <td colspan="8" align="center">
+ *                                         0/1 - the value affects only
+ *                                         registration/mapping phase </td>
+ * <tr><td>@ref UCP_MEM_MAP_ALLOCATE       <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ * <tr><td>@ref UCP_MEM_MAP_FIXED          <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">1</td>
+ * <tr><td>@ref ucp_mem_map_params.address <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">defined</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">defined</td>
+ *                                         <td align="center">defined</td>
+ *                                         <td align="center">defined</td>
+ * <tr><td>Result                          <td>@ref anch_err "error"
+ *                                         <td>@ref anch_alloc_reg "alloc+register"
+ *                                         <td>@ref anch_err "error"
+ *                                         <td>@ref anch_reg "register"
+ *                                         <td>@ref anch_err "error"
+ *                                         <td>@ref anch_alloc_hint_reg "alloc+register,hint"
+ *                                         <td>@ref anch_err "error"
+ *                                         <td>@ref anch_alloc_fixed_reg "alloc+register,fixed"
+ * </table>
```

Comment:
@bbenton formatting for PDF is a pain.
The table is too wide without abbreviations even in ASCII and it doesn't support hyperlinks to provide good description and navigation.
I've prepared 3 different variants of the table, please take patch [here](https://gist.github.com/evgeny-leksikov/4cbb0d1ace367de5ea82a20e596903dd) and choose the best one from your point of view. If you would play with it and provide better formatting, it would be very appreciated!